src/components/TodoList.scss

```diff
@@ -0,0 +1,15 @@
+.TodoList {
+  max-height: 400px;
```

Comment:
Don't add height limit, to disable vertical scrolling your list

Reply:
Thanks!